### PR TITLE
Flaky test fix - make archive file random

### PR DESCRIPTION
**Pull request by brybacki**
Signed-off-by: Bartosz Rybacki <brybacki@redhat.com>

**What this PR does / why we need it**:

The archive file was reused in many tests. Tests were 
failing when multiple tests created and deleted it at the 
same time (parallel tests). Fix is easy - create a 
temporary file with random name.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

https://github.com/kubevirt/kubevirt/issues/7203

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ShellyKa13** [comment]
/lgtm
Thanks for spotting this :)

File: pkg/virtctl/imageupload/imageupload_test.go [hunk 1]
```diff
@@ -77,7 +77,12 @@ var _ = Describe("ImageUpload", func() {
 
 		imagePath = imageFile.Name()
 
-		archiveFilePath = tests.ArchiveFiles("archive", os.TempDir(), imagePath)
+		archiveFile, err := ioutil.TempFile("", "archive")
+		Expect(err).ToNot(HaveOccurred())
+		defer archiveFile.Close()
```
Comment (mhenriks) [inline pkg/virtctl/imageupload/imageupload_test.go:82]: Does this file get deleted?

Comment (brybacki) [reply pkg/virtctl/imageupload/imageupload_test.go:82]: in `AfterEach(func() {` line 91: `os.Remove(archiveFilePath)`

**Pull request merged**